website/docs/reference/resource-configs/grants.md

```diff
@@ -98,152 +72,117 @@ The `meta` config can also be defined under the `seeds` config block in `dbt_pro
 <File name='snapshots/schema.yml'>
 
 ```yml
-version: 2
-
 snapshots:
   - name: snapshot_name
-    config:
-      meta: {<dictionary>}
-
-    columns:
-      - name: column_name
-        meta: {<dictionary>}
-
+    config:  
+      grants:
+        select: ['reporter', 'bi']
 ```
 
 </File>
 
-The `meta` config can also be defined:
+The `grants` config can also be defined:
+
 - under the `snapshots` config block in `dbt_project.yml`
 - in a `config()` Jinja macro within a snapshot's SQL block
 
 See [configs and properties](configs-and-properties) for details.
 
 </TabItem>
+</Tabs>
 
-<TabItem value="analyses">
-
-The `meta` config is not currently supported for analyses.
-
-</TabItem>
+## Definition
 
-<TabItem value="macros">
+You can use the `grants` field to set permissions or grants for a resource. These grants will be compiled into the `manifest.json` file complied by dbt.
 
-<File name='macros/schema.yml'>
+## Database-specific requirements and notes
 
-```yml
-version: 2
+While we try to standardize the terms we use to describe different features, you will always find nuances in different databases. This section outlines some of those database-specific requirements and notes.
 
-macros:
-  - name: macro_name
-    meta: {<dictionary>}
+### Common syntax 
 
-    arguments:
-      - name: argument_name
-        meta: {<dictionary>}
+In our examples, you find terms like `select` and `another_user` because many databases use these terms, but be aware of the syntax your own database supports:
 
-```
+* Privileges: A right to perform an action in a database.
+* Grantees: A way to manage privileges. Recipients of granted privileges, also called "principals." Grantees can be a user, a group of users, a role held by users (Snowflake), a service account (GCP), and more.
 
-</File>
+<WHCode>
 
-</TabItem>
+<div warehouse="BigQuery">
 
-<TabItem value="exposures">
+- Be aware that the `grants` config is unrelated to the `grant_access_to` config:
+  - **`grants_access_to`:** Enables you to set up authorized views. When configured, dbt provides authorized view access to other datasets, without leaking additional data from those datasets. Fore more on this, see [BigQuery configurations: Authorized views](/reference/resource-configs/bigquery-configs#authorized-views)
+  - **`grants`:** Provides specific permissions to users, groups, or service accounts for managing access to datasets you're producing with dbt.You could grant a user, group or service account access to an authorized view set up by the `grants_access_to` feature.
```

Comment:
```suggestion
- **`grants`:** Provides specific permissions to users, groups, or service accounts for managing access to datasets you're producing with dbt.

You can use the two features together: "authorize" a view model with the `grants_access_to` configuration, and then add `grants` to that view model to share its query results (and _only_ its query results) with other users, groups, or service accounts.
```
What this looks like in code, if helpful:
```sql
{{ config(
  materialized = 'view',
  grant_access_to = [{'project': 'sensitive-project', 'dataset': 'sensitive-dataset'}],
  grants = {'roles/bigquery.dataViewer': ['group:analysts@yourcompany.com']}
) }}

select * from {{ ref('model_with_sensitive_data') }} 
where is_sensitive_data = False
```
Big idea: members of the Google group `analysts@yourcompany.com` get to see the data included in this view, but they _don't_ get access to the underlying data in the sensitive model / dataset